packages/blockchain-wallet-v4-frontend/src/modals/BuySell/AddCardCheckoutDotCom/template.success.tsx

```diff
@@ -90,7 +90,7 @@ const Success = ({
             />
           </Expanded>
 
-          <Card backgroundColor='grey000' borderRadius='md'>
+          <Card backgroundColor='grey-000' borderRadius='md'>
```

Comment:
This decision can bite us in the future. Why every component needs the actual `PaletteColors[key]` but this? Feels inconsistent with the rest of the Library somehow